libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -144,6 +145,16 @@ func UseSystemd() bool {
 			}
 		}
 
+		hasDelegate = true
+		dl := newProp("Delegate", true)
+		if _, err := theConn.StartTransientUnit(scope, "replace", []systemdDbus.Property{dl}, nil); err != nil {
```

Comment:
Don't you have to destroy this unit first? Or is this dealt with by the "stop unit logic"?